Dynamic programming - algos/binomial_coeff.cpp

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
@sinhatushar Please use `/*` for start comments in multiple lines and `*/` for ending.And please also make these code to prompt the user to ask what he has to enter:)

Reply:
Changes have been made as per your and @22shubh22 's review.